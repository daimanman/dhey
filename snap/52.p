{"C":1,"Duration":63670602308,"EndTime":"2018-08-23T14:25:08.5692955+08:00","Err":"","FileId":"4","Method":"POST","N":1,"P":{"info":"TEssssssssskkkkkk颠三倒四ST","name":"sssssDXwwM"},"Payload":"","QH":{},"Remark":"压力测试","ReqNums":1,"StartTime":"2018-08-23T14:25:08.5578221+08:00","Status":1,"TaskId":52,"Type":"FIXED","Url":"http://localhost:9188/go1","Z":""}