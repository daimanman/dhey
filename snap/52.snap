{"AvgTotal":0.0114734,"Fastest":0.0114734,"Slowest":0.0114734,"Average":0.0114734,"Rps":87.15812226541391,"AvgConn":0.0057064,"AvgDNS":0.0057064,"AvgReq":0,"AvgRes":0.0005073,"AvgDelay":0.0029925,"ConnMax":0.0057064,"ConnMin":0.0057064,"DnsMax":0.0057064,"DnsMin":0.0057064,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0.0005073,"DelayMax":0.0029925,"DelayMin":0.0029925,"Lats":[0.0114734],"ConnLats":[0.0057064],"DnsLats":[0.0057064],"ReqLats":[0],"ResLats":[0.0005073],"DelayLats":[0.0029925],"Total":11473400,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":62,"SizeReq":62,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.0114734,"Count":1,"Frequentcy":1},{"Mark":0.0114734,"Count":0,"Frequentcy":0},{"Mark":0.0114734,"Count":0,"Frequentcy":0},{"Mark":0.0114734,"Count":0,"Frequentcy":0},{"Mark":0.0114734,"Count":0,"Frequentcy":0},{"Mark":0.0114734,"Count":0,"Frequentcy":0},{"Mark":0.0114734,"Count":0,"Frequentcy":0},{"Mark":0.0114734,"Count":0,"Frequentcy":0},{"Mark":0.0114734,"Count":0,"Frequentcy":0},{"Mark":0.0114734,"Count":0,"Frequentcy":0},{"Mark":0.0114734,"Count":0,"Frequentcy":0}],"TaskId":52,"Duration":0.0114734}